mlir/lib/Dialect/Vector/Transforms/LowerVectorMultiReduction.cpp

```diff
@@ -334,11 +341,15 @@ struct TwoDimMultiReductionToElementWise
     for (int64_t i = 0; i < srcShape[0]; i++) {
       auto operand = rewriter.create<vector::ExtractOp>(
           loc, multiReductionOp.getSource(), i);
+      Value extractMask = nullptr;
+      if (mask) {
+        extractMask = rewriter.create<vector::ExtractOp>(loc, mask, i);
+      }
       result = makeArithReduction(rewriter, loc, multiReductionOp.getKind(),
-                                  operand, result);
+                                  operand, result, nullptr, extractMask);
```

Comment:
-> /* nameOfTheFuncParam=nullptr */ for readability

Reply:
done